src/containers/Settings/components/General.jsx

```diff
@@ -289,6 +309,49 @@ class Settings extends React.Component {
             </Button>
           </CardActions>
         </Card>
+
+        <Card className={css(styles.sectionCard)}>
```

Comment:
Suggestion: I think our preference is for new Cards to be extracted as separate files ([examples in lines further down](https://github.com/politics-rewired/Spoke/blob/main/src/containers/Settings/components/General.jsx#L418-L426)). You can also use MUI v4 Cards that way 😄